static/js/lib/form.js

```diff
@@ -44,3 +44,20 @@ export const formatErrors = (
   // $FlowFixMe
   return <div className="error">{errorString}</div>
 }
+
+export const formatMessages = (
```

Comment:
It is not necessary to copy the logic of `formatErrors` here. Right now we only have one client of this function, it only passes a string, and there is no reason to believe that our API might return some kind of object that we will extract success messages from. Please simplify this and rename it:

```javascript
export const formatSuccessMessage = (
  message: string
): React$Element<*> => {
  // $FlowFixMe
  return <div className="success">{message}</div>
}